docs/src/integrations/exchange.md

```diff
@@ -488,6 +488,145 @@ You can also doublecheck whether a particular blockhash is still valid by sendin
 request with the blockhash as a parameter. If the response value is null, the
 blockhash is expired, and the withdrawal transaction should never succeed.
 
+## Supporting the SPL Token Standard
+
+[SPL Token](https://spl.solana.com/token) is the standard for wrapped/synthetic
+token creation and exchange on the Solana blockchain.
+
+The SPL Token workflow is similar to that of native SOL tokens, but there are a
+few differences which will be discussed in this section.
+
+### Token Mints
+
+Each *type* of SPL Token is declared by creating a *mint* account.  This account
+stores metadata describing token features like the supply, number of decimals and
+various authorities with control over the mint.  Each SPL Token account references
+its associated mint and may only interact with SPL Tokens of that type.
+
+### Installing the `spl-token` CLI Tool
```

Comment:
I find the ordering and heading levels to be a bit weird and hard to follow, especially this fairly lengthy section on installing a tool that I don't yet know the need for. Could we shunt the installation information to somewhere different and just link to it under `## Supporting the SPL Token Standard` above? (Something like `Example commands below utilize the SPL Token CLI toll, which can be installed by follow the steps [here](link)`)

Reply:
Yeah I want to clean up https://spl.solana.com/token then link there from here.  It's not ready yet though, so I figured we dupe here to get integration unblocked.

Would an introductory sentence here like, "SPL Token accounts are managed with the `spl-token` command line tool"

Reply:
https://github.com/solana-labs/solana/pull/12303/commits/5141cad6244ecec0940cfd6f3abdd893ec80309e

Reply:
All right, works for now